draft-ietf-cose-cwt-claims-in-headers.md

```diff
@@ -69,6 +69,8 @@ CWT-Claims = {
 Claim-Label = int / text
 ```
 
+It is RECOMMENDED that the cwt claims header parameter is used only in a protected header to avoid the contents from being malleable. The header parameter MUST only occur once in either the protected or unprotected header of a COSE structure.
```

Comment:
cwt -> CWT (and do this everywhere else in the draft that it's currently lowercase)